src/modules/lib/LinkedIdList.sol

```diff
@@ -9,13 +9,16 @@ Assumptions
 */
 
 library LinkedIdList {
+    //--------------------------------------------------------------------------
+    // Structs
+
+    /// @param Size of the list.
+    /// @param Marks the last element of the list.
+    /// Always links back to the _SENTINEL.
```

Comment:
Looks a bit lost again this line, either add to the one above or something else, not sure.